Docker 8/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM tomcat:latest
+
+RUN cd /usr/local/tomcat && \
+    mv webapps webapps2 && \
+    mv webapps.dist webapps
+
+EXPOSE 8080
```

Comment:
no need to write it again, we get all the other commands from the "FROM" command